libs/blocks/tabs/tabs.css

```
@@ -468,6 +468,17 @@
   margin-block-end: var(--spacing-xs);
 }
 
+.tabs.radio div[role="tablist"] button,
+.tabs.radio div[role="tablist"] button:hover,
+.tabs.radio div[role="tablist"] button:focus-visible,
+.tabs.radio div[role="tablist"] button[aria-selected="true"] {
```

Let's trim down some repetition and use `:is`
```suggestion
.tabs.radio div[role="tablist"] :is(button, ...) {
```